spec/menu_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'menu'
+
+describe Menu do
+  it "diplays menu items" do
+    expect { subject.show }.
+    to output("1. BBQ: £10\n2. Hamburger: £8\n" +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.<br>Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.